src/Broker/BrokerFactory.php

```diff
@@ -21,6 +21,7 @@ class BrokerFactory
 	public const DYNAMIC_METHOD_RETURN_TYPE_EXTENSION_TAG = 'phpstan.broker.dynamicMethodReturnTypeExtension';
 	public const DYNAMIC_STATIC_METHOD_RETURN_TYPE_EXTENSION_TAG = 'phpstan.broker.dynamicStaticMethodReturnTypeExtension';
 	public const DYNAMIC_FUNCTION_RETURN_TYPE_EXTENSION_TAG = 'phpstan.broker.dynamicFunctionReturnTypeExtension';
+	public const OPERATOR_TYPE_SPECIFYING_EXTENSION_TAG = 'phpstan.broker.operatorTypeSpecifyingExtension';
```

Comment:
This needs to be added to `ConditionalTagsExtension::getConfigSchema()` too.

Reply:
Could you tell me more about it? I'm not sure what to do ;)

Reply:
This tag needs to be added there just as the others are. A single line...

Reply:
I used wrong remote during rebase, now everything is clear :)